doc/devices/wavefunction.rst

```diff
@@ -95,4 +64,20 @@ These operations include:
 .. autosummary::
     pennylane_forest.ops.CPHASE
     pennylane_forest.ops.ISWAP
-    pennylane_forest.ops.PSWAP
\ No newline at end of file
+    pennylane_forest.ops.PSWAP
+
+QVM and quilc server configuration
+~~~~~~~~~~~~~~
+
+.. note::
```

Comment:
Given that this is repeated, can we write it one place and just import/reference it in the places it's used?

Reply:
Done